### PR TITLE
Contrast 34726 release notes

### PR DESCRIPTION
**Pull request by kaitlynmortimer**
Release notes for 3.6.5

https://bitbucket.org/contrastsecurity/open-docs/pull-requests/318/contrast-34726-release-notes/diff

### CODE REVIEW

File: content/release/ReleaseNotes365.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+### Ruby summary 
+
+The Ruby team delivered expanded rule coverage and better performance for our 2.6.0 release. The agent now supports an Unsafe File Upload Protect rule to block attacks at perimeter as well as Server Side Request Forgery (SSRF) detection in Assess. We also enhanced our string instrumentation rewriting along with other under-the-hood performance improvements.
+
+### Python summary
+
+The Python agent's 1.10.0 release introduces a few important changes. We're dropping support for Python 3.4 as it's reached its End of Life date. We also allow users to set Protect rule modes in the [configuration YAML](installation-pythonconfig.html), which gives you more control over deployed instances of the agent without using the Contrast UI. We also improved response handling on SecurityExceptions: In the instance application code catches our exception during an attack, the agent will send the exception after application code has completed, if we need to block a request. Some minor bug fixes include the CSRF header used by the agent and improvements for the PyramidMiddleware with legacy Pyramid versions.
+
```
Comment (kaitlynmortimer) [inline content/release/ReleaseNotes365.md]: @jleo32 The original copy says, "...after application code has **completely** if we need to block a request." Is this edit correct?

Comment (jleo32) [reply content/release/ReleaseNotes365.md]: completed*, oops.

File: content/release/ReleaseNotes365.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+## Agent Updates
+
+### Java summary
+
+Check back next release for updates!
+
+### .NET summary 
+
+The .NET team added support for agent use within [Docker containers](installation-netinstall.html#net-docker) for .NET 4.5.2+ applications. We improved accuracy of an insecure authentication protocol Assess rule, and added a new Assess rule to detect the “X-Powered-By” header. We fixed a bug that caused a null reference error in processing exclusions. We also added support for setting `application.code`.
```
Comment (DanFiedler) [inline content/release/ReleaseNotes365.md]: We need to add a note about removing support for two legacy configuration options. Here's my attempt:

> The legacy DotnetAgentService.exe.config is no longer supported for application pool whitelisting/blacklisting, please move these configuration values to `agent.dotnet.app_pool_whitelist` and `agent.dotnet.app_pool_blacklist` in contrast_security.yaml. This change applies to all versions after 19.5.4.

File: content/release/ReleaseNotes365.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+### .NET Core summary
+
+The [.NET Core agent](installation-netcore.html#netcore-overview) for Windows is now available! The agent supports the same expansive Assess and Protect security policy as the .NET Framework agent, just for your .NET Core applications. To start using the agent, check out the [system requirements](installation-netcore.html#netcore-system), and then [download and install](installation-netcoreinstall.html) the agent from the Contrast UI.
```
Comment (DanFiedler) [inline content/release/ReleaseNotes365.md]: The .NET Core agent currently has a slightly smaller policy / set of supported rules than the full .NET Framework agent. (We're working to fill in the gaps...) I'm not sure the best way to word this but I'd be comfortable saying that the .NET Core agent supports most of the same policy as the .NET framework agent, including detection of all of the most important vulnerabilities and attacks.

**Pull request merged**